Support text selection / clickable UI for readonly objectbrick fields

BlackbitDevs opened the pull request:
Similar to #190 when you set an object brick field to `not editable` it gets completely disabled. Thus you can neither select any text nor can you click buttons to jump to relations (or other UI actions).

Without this PR:
<img width="771" alt="Bildschirmfoto 2023-08-01 um 17 36 46" src="https://github.com/pimcore/admin-ui-classic-bundle/assets/8749138/0c837610-a3fa-4288-bed0-7c249384cd1b">

With this PR:
<img width="729" alt="Bildschirmfoto 2023-08-01 um 17 36 19" src="https://github.com/pimcore/admin-ui-classic-bundle/assets/8749138/ae870a5d-40d0-4ccd-b85c-d92db18d2d46">

Commit:
support text selection for readonly objectbricks fields

kingjia90 commented:
Thank you for the PR! 
Closing in favor of https://github.com/pimcore/pimcore/pull/15720 as this PR is changing behaviour/display (grayed zone) and was actually a potential bug for 10.6

kingjia90 commented on public/js/pimcore/object/tags/objectbricks.js:
```diff
@@ -300,17 +300,12 @@ pimcore.object.tags.objectbricks = Class.create(pimcore.object.tags.abstract, {
-                                children.forEach(function (record) {
-                                    record.disabled = true;
-                                });
-                            }
```
to keep the grayed mask, i've added here
```
if (this.fieldConfig.noteditable){
    panel.mask();
}
```
and a css rule as 

```
.pimcore_objectbrick_item.x-masked .x-mask{
    pointer-events: none;
}
```
to make it work without visual changes, could you please also verify? 

Then it should go on 10.6 as well. Let me know if you can do that before the end of the day, because i may have to takeover as tomorrow would be the last 10.6 release

kingjia90 replied:
Follow-up note: this suggestion to add a base `mask` is meant to be only valid for 10.6 to avoid the display changes, in 11 it can be considered to be removed https://github.com/pimcore/pimcore/pull/15759#issue-1854317248

kingjia90 replied:
the removal of the mask would be postponed a little bit and may take a confirmation first, merging this PR as the equivalent/port of https://github.com/pimcore/pimcore/pull/15759 for Pimcore 11

kingjia90 committed:
revert the change to add mask

see https://github.com/pimcore/pimcore/pull/15759/commits/e85d60ac47aa52ad3c2316dde38d8d27b2c8d670

kingjia90 committed:
fix css

same as https://github.com/pimcore/pimcore/pull/15759/commits/78064d9a6e0ba36a5c434ac473de59ce4220a1c4
